ui/src/App.tsx

```diff
@@ -65,12 +62,20 @@ const App = () => {
   return (
     <div className="App">
       <div>
-        <Calendar
-          localizer={localizer}
+        <FullCalendar
+          plugins={[dayGridPlugin, timeGridPlugin, listPlugin]}
+          headerToolbar={{
+            left: "prev,next today",
+            center: "title",
+            right: "dayGridMonth,timeGridWeek,timeGridDay",
+          }}
           events={events}
-          startAccessor="start"
-          endAccessor="end"
-          style={{ height: 500 }}
+          initialView="dayGridMonth"
+          // editable={true}
+          // selectable={true}
+          // selectMirror={true}
+          // dayMaxEvents={true}
+          // weekends={true}
```

Comment:
Shall we remove these comments? Or are they intended as documentation of possible props?

Reply:
Intended as documentation / for future use. Planning to leave for now.